.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
     - $HOME/.cache/pip
     - $TRAVIS_BUILD_DIR/war
 
-script: ./run_tests.sh
+script: python ./run_tests.py
```

Comment:
I would make run_tests.py executable by the user and run it directly. We have the shebang set in that file so the bash interpreter knows which application to use.